modules/hardware/nvidia-jetson-orin/random-seed

```diff
@@ -0,0 +1 @@
+���V��o->�z"Gv?qQ���Ҳ=�:(:V�
```

Comment:
From the base commit, I decided to leave the random-seed out because it seems to work even without it